### ISSUE TITLE
Hello, will you have any support for linux?

### ISSUE THREAD

**Issue by tridoxx** [IceBeach7/OpenRGB-Updater#2]
Hello, I would like to know if you suddenly create an auto updater for linux, thank you very much

**Comment by IceBeach7** [comment]
Hi,
Linux support would certainly be possible. We would have to make [several versions](https://gitlab.com/CalcProgrammer1/OpenRGB#linux) for different distributions, as AppImages are only provided on stable releases of OpenRGB. OpenRGB is also on [Flathub](https://flathub.org/apps/details/org.openrgb.OpenRGB), so maybe a plugin manager would be sufficient for Linux.

**Comment by tridoxx** [comment]
@IceBeach7 
Yes, thank you very much, although it also depends on how long the script is, because you could build it for debian-based OS as a start, and for my part I could do a translation for arch linux (I used arch btw), now How do you know those based on Debian are the most used for gaming etc.

**Comment by IceBeach7** [comment]
@tridoxx
It should work on any distro now. I tried it on Ubuntu 22.10.
Could you try it out on Arch, so we can merge it into main?
